pyworkflow/protocol/params.py

```diff
@@ -648,8 +647,8 @@ def __init__(self, error='Boolean param needs to be set.'):
 
 Positive = GT(0.0, error='Value should be greater than zero')
 
-FreqValidator = Range(0., 0.5, 
-                      error="Digital frequencies should be between 0. and 0.5")
+FreqValidator = Range(0.001, 0.5,
```

Comment:
To me this change seems logic....any reason to keep 0.?
@delarosatrevin , @rmarabini

Reply:
I think I found this is only used in spider plugin, didn't check xmipp though